pkg/apis/agones/v1/gameserver_test.go

```diff
@@ -319,8 +326,15 @@ func TestGameServerApplyDefaults(t *testing.T) {
 		},
 	}
 
+	runtime.FeatureTestMutex.Lock()
+	defer runtime.FeatureTestMutex.Unlock()
+
+	// otherwise the race condition detector is not happy.
```

Comment:
Otherwise refers to what here?

Reply:
That should actually go away. That was leftover from a previous locking strategy attempt.